src/wallet_new.ts

```diff
@@ -78,6 +81,15 @@ export class WalletDAO {
 		return Promise.resolve(+(await this.arweave.wallets.getBalance(address)));
 	}
 
+	async confirmBalance(wallet: Wallet, winstonPrice: Winston): Promise<boolean> {
+		const walletBalance = await this.getWalletWinstonBalance(wallet);
+		return +walletBalance > +winstonPrice;
+	}
+
+	async getWinstonPriceForBytes(bytes: Bytes): Promise<Winston> {
```

Comment:
Might want to add a TODO here to use the price estimator utility when once we reintegrate it with core.

Reply:
Also I'm not sure this should live in the WalletDAO. But I don't think there's a better spot for it in the hierarchy at the moment.